examples/basic/modal_acoustics_analysis.py

```diff
@@ -0,0 +1,553 @@
+""" .. _ref_modal_acoustics_analysis:
+
+Modal acoustics analysis
+------------------------
+
+This example demonstrate modal acoustic analysis that involves
+modeling both a structure and the surrounding
+fluid to analyze frequencies and standing wave patterns within the structure.
+This type of analysis is essential for applications such as Sonar, concert hall design,
+noise reduction in various settings, audio speaker design, and geophysical exploration.
+
+Mechanical enables you to model pure acoustic problems and fluid-structure
+interaction (FSI) problems.A coupled acoustic analysis accounts for FSI.
+An uncoupled acoustic analysis simulates
+the fluid only and ignores any fluid-structure interaction.
```

Comment:
Mechanical enables you to model pure acoustic problems and fluid-structure interaction (FSI) problems. Coupled acoustic analyses account for FSI. An uncoupled acoustic analysis simulates the fluid only and ignores any fluid-structure interaction. This example demonstrates a modal acoustic analysis that models a structure as well as the surrounding fluid in order to analyze frequencies and standing wave patterns within the structure. This type of analysis is essential for applications such as Sonar, concert hall design, noise reduction in various settings, audio speaker design, and geophysical exploration.